lerobot/scripts/eval.py

```diff
@@ -342,11 +350,13 @@ def render_frame(env: gym.vector.VectorEnv):
         # Maybe render video for visualization.
         if max_episodes_rendered > 0 and len(ep_frames) > 0:
             batch_stacked_frames = np.stack(ep_frames, axis=1)  # (b, t, *)
+            assert isinstance(videos_dir, Path)
```

Comment:
Perhaps we should check this at the very start, and raise a `ValueError` if the caller tries to provide `max_episodes_rendered > 0` without `videos_dir`. What do you think?